homeassistant/components/sleepiq/number.py

```diff
@@ -120,6 +144,18 @@ def _get_foot_warming_unique_id(bed: SleepIQBed, foot_warmer: SleepIQFootWarmer)
         get_name_fn=_get_actuator_name,
         get_unique_id_fn=_get_actuator_unique_id,
     ),
+    CORE_CLIMATE_TIMER: SleepIQNumberEntityDescription(
+        key=CORE_CLIMATE_TIMER,
+        native_min_value=30,
+        native_max_value=360,
+        native_step=30,
+        name=ENTITY_TYPES[CORE_CLIMATE_TIMER],
+        icon="mdi:timer",
+        value_fn=lambda core_climate: core_climate.timer,
```

Comment:
What does this value represent? The total time? Or the time left (and thus counting down)?

Reply:
It's a count down that is set-able and then is a countdown. Same implementation as previously implemented for the feet warming feature.

Reply:
It also has default value set when climate it activated. That's all managed on the sleep number side.